qa/logging-config/build.gradle

```diff
@@ -25,7 +25,10 @@ apply plugin: 'elasticsearch.standalone-test'
 integTestCluster {
     autoSetInitialMasterNodes = false
     autoSetHostsProvider = false
-    // overriding the distribution log4j configuration file with a custom configuration
+    /**
+     * The intention is to provide a custom log4j configuration where layout is an old style pattern and confirm that Elasticsearch 
+     * can successfully startup. see CustomLoggingIT.
```

Comment:
It is not `CustomLoggingIT` but `CustomLoggingConfigIT`. Anyway, I don't think we need to mention it?